src/index.css

```
@@ -1,6 +1,10 @@
-@import url('https://cdn.jsdelivr.net/npm/open-props@1.0.0/open-props.min.css');
+@import "open-props/style";
+@import "open-props/open-props.min.css";
```

```suggestion
@import "open-props/open-props.min.css";
```

This is a redundant import, let's use the direct css file one.